samples/api/README.adoc

```diff
@@ -90,6 +94,14 @@ A transcoded version of the API sample <<timestamp_queries,Timestamp queries>> t
 
 Uses the instancing feature for rendering many instances of the same mesh from a single vertex buffer with variable parameters and textures.
 
+=== xref:./{api_samplespath}oit_depth_peeling/README.adoc[OIT Depth Peeling]
```

Comment:
These two OIT samples are already listed here, see the end of the readme. This would be a duplicate.

Reply:
Oops... missed them.
I thought, the samples were alphabetically ordered here.